### DISCUSSION TITLE
ADR Suggestion: Standardize on Python version support strategies

### DISCUSSION THREAD

**Discussion by rozyczko** [easyscience/.github#26] [Ideas]
We should have a way to determine when to stop supporting older python versions and when to start supporting new versions.

This is to a large extent dependent on our main dependency - scipp.

We could possibly mirror scipp's support management for Python versions and important packages like numpy, scipy etc.

However, there is a non-trivial issue with modules like PySide (and to a lesser degree - pyinstaller), which are very cranky wrt. version dependencies.

**Comment by rozyczko** [comment]
We need to also consider requirements for the calculators we use in our projects
- cryspy
- pycrysfml

**Comment by damskii9992** [comment]
Is this really a corelib discussion or an organization-level discussion? And did we agree to follow the version-support of Scipp?

**Comment by rozyczko** [comment]
This should be an org-level discussion.
We did discuss following scipp for version support, at least for main modules like python/numpy.

**Comment by damskii9992** [comment]
Moved the discussion to the organization level and will close it and make a proper ADR
